### DISCUSSION TITLE
增大PDF viewer渲染的字重

### DISCUSSION THREAD

**Discussion by Theigrams** [MuiseDestiny/zotero-style#453] [Q&A] [answered]
您好，本人使用环境是M1 Mac，Zotero6
Zotero6内置PDF阅读器渲染的字体有点虚，影响阅读体验，因此想借助PDF style通过修改CSS改变字体的粗细/字重。
请问PDF style能实现吗？如果可以的话改怎么修改呢？
谢谢~

<img width="1460" alt="image" src="https://github.com/MuiseDestiny/zotero-style/assets/26341831/ee999c9b-d1af-41a2-b668-063a34eca742">
图1. Zotero内置PDF浏览器
<img width="1566" alt="image" src="https://github.com/MuiseDestiny/zotero-style/assets/26341831/d321c753-8ec8-48f4-8f53-4afc025f5ca8">
图2. Skim PDF浏览器

**Comment by MuiseDestiny** [comment, marked as answer]
不能，这个官方也不知道怎么修复。不是简单改css就能解决的。应该与火狐这个平台有关。

**Reply by Theigrams** [reply]
好的，感谢解惑

**Comment by MuiseDestiny** [comment]
就是同样的pdf用火狐打开也会虚，越是放大越明显。
